tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -520,13 +517,13 @@ void OpTest::dispatchTrigonometricOpTestByDataType(
         DataType.c_str());
 }
 
-template <typename DataTypeT, typename OpTypeT>
-void OpTest::dispatchTestByVectorLength(const OpTypeMetaData<OpTypeT> &OpTypeMd,
+template <typename T, typename T2>
+void OpTest::dispatchTestByVectorLength(const OpTypeMetaData<T2> &OpTypeMd,
```

Comment:
IMO we've lost a bit too much here and could do with more descriptive names when there are multiple template parameters.  Especially since `T` is more of the "standard numeric type that we'll be doing tests on" while `T2` is more "one of the enums that represent operations".

Maybe `T` and `OP` would be ok?

Reply:
How about T and OpT?